Index: resources/usecases/Player moves.puml
===================================================================
diff --git a/resources/usecases/Player moves.puml b/resources/usecases/Player moves.puml
new file mode 100644
--- /dev/null	
+++ b/resources/usecases/Player moves.puml	
@@ -0,0 +1,24 @@
+@startuml
+
+left to right direction
+skinparam packageStyle rectangle
+
+actor Player as P
+rectangle "Game" {
+  usecase "Change Room" as CR
+  usecase "Check Room Availability" as CRA
+  usecase "move" as mv
+  usecase "check if there's teacher" as CT
+}
+
+P --> CR : Request Room Change to <Room>
+CRA .> CR : <<includes>>
+CRA <. CT : [if available]
+CT <. mv : [if no teacher]
+CT --> P : you died
+
+mv --> P : Moved to <Room>
+CRA --> P : Room full
+
+
+@enduml
\ No newline at end of file
