Index: resources/usecases/Player moves.puml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/resources/usecases/Player moves.puml b/resources/usecases/Player moves.puml
new file mode 100644
--- /dev/null	(date 1710686308470)
+++ b/resources/usecases/Player moves.puml	(date 1710686308470)
@@ -0,0 +1,24 @@
+@startuml
+
+left to right direction
+skinparam packageStyle rectangle
+
+actor Player as P
+rectangle "Game" {
+  usecase "Change Room" as CR
+  usecase "Check Room Availability" as CRA
+  usecase "move" as mv
+  usecase "check if there's teacher" as CT
+}
+
+P --> CR : Request Room Change to <Room>
+CRA .> CR : <<includes>>
+CRA <. CT : [if available]
+CT <. mv : [if no teacher]
+CT --> P : you died
+
+mv --> P : Moved to <Room>
+CRA --> P : Room full
+
+
+@enduml
\ No newline at end of file
Index: .idea/.github.iml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<module type=\"JAVA_MODULE\" version=\"4\">\r\n  <component name=\"NewModuleRootManager\" inherit-compiler-output=\"true\">\r\n    <exclude-output />\r\n    <content url=\"file://$MODULE_DIR$\" />\r\n    <orderEntry type=\"inheritedJdk\" />\r\n    <orderEntry type=\"sourceFolder\" forTests=\"false\" />\r\n  </component>\r\n  <component name=\"SonarLintModuleSettings\">\r\n    <option name=\"uniqueId\" value=\"5efe14e7-adbe-478f-84d8-189d5f726eab\" />\r\n  </component>\r\n</module>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/.github.iml b/.idea/.github.iml
--- a/.idea/.github.iml	(revision 6f7f200a9f560789215d7ef1a8e4732b31683dc3)
+++ b/.idea/.github.iml	(date 1710686308481)
@@ -9,4 +9,7 @@
   <component name="SonarLintModuleSettings">
     <option name="uniqueId" value="5efe14e7-adbe-478f-84d8-189d5f726eab" />
   </component>
+  <component name="SonarLintModuleSettings">
+    <option name="uniqueId" value="dee3f5ad-6da6-4332-9d74-277b58c8e6c0" />
+  </component>
 </module>
\ No newline at end of file
